### PR TITLE
Remove keen slider for carousel and create our own carouse component

### PR DESCRIPTION
**Pull request by bangunbagustapa**
### Description

There's an issue with [keen-slider](https://keen-slider.io/) when it renders in the client-side. The library isn't working well with hybrid rendering.

Why not use another library instead? Most slider / carousel library for Vue.js work for client-side rendering, there's a few library that support server-side rendering, like [keen-slider](https://keen-slider.io/) but not hybrid rendering.

### Solution

We create our own simple carousel to provide both client-side rendering and server-side rendering. These change spesific for our current use case.

### Changes

- Create carousel component
- Replace all component that used old carousel to the new one

### Evidence

Project: Portal Jabar
Title: Remove keen slider for carousel and create our own carouse component
Participants: @bangunbagustapa @Ibwedagama @adzharamrullah @naufalihsank

### CODE REVIEW

**Review by Ibwedagama — APPROVED** [review]

File: components/News/Header/index.vue [hunk 1]
```diff
@@ -72,26 +66,26 @@
                   Berita Terkait
                 </p>
                 <div class="flex flex-col gap-2">
-                  <Link v-for="(news, idx) of item.related_news" :key="news.id" :link="`/berita/${getRelatedNewsSlug(slider, idx)}`">
-                    <div class="flex gap-4 p-2 bg-white bg-opacity-0 rounded-xl">
+                  <Link v-for="news of item.related_news.slice(0, 4)" :key="news.id" :link="`/berita/${news.slug}`" class="group">
+                    <div class="flex newss-center gap-4 p-2 bg-white bg-opacity-0 group-hover:bg-opacity-5 rounded-xl">
```
Comment (adzharamrullah) [inline components/News/Header/index.vue:70]: I think you're a typo write `newss-center`

Comment (Ibwedagama) [inline components/News/Header/index.vue:70]: is this `newss-center` class a typo? @bangunbagustapa

Comment (bangunbagustapa) [reply components/News/Header/index.vue]: Yes, it shouldn't be there. Fixed [here](https://github.com/jabardigitalservice/portal-jabar-frontend/pull/92/commits/f0566102bbc79b99056e9bc601d88d09e672c7d3), thank you!

Comment (bangunbagustapa) [reply components/News/Header/index.vue]: Yes, it shouldn't be there. Fixed [here](https://github.com/jabardigitalservice/portal-jabar-frontend/pull/92/commits/f0566102bbc79b99056e9bc601d88d09e672c7d3), thank you!

**Review by naufalihsank — APPROVED** [review]

**Review by adzharamrullah — APPROVED** [review]

**Pull request merged**